Troubleshooting: Component Content is not Scrollable (#5442)
DevExpress/devextreme-documentation: concepts/80 Troubleshooting/45 Component Content is not Scrollable/00 Component Content is not Scrollable.md

> This topic covers issues related to components with scrollable content: DataGrid, TreeList, Scheduler, Form, ScrollView, Popup, etc. The following issues may occur:

Reviewer: "Covers" is not the best word to use. I think Ray doesn't really like it. If you already used "describes" or "explains" and need a synonym to not be repetitive, then maybe it's okay. But here I would make a change. 

I also want to emphasize that issues are related to "scrollable content", not "components". 

```suggestion
This topic describes issues related to scrollable content in various components: DataGrid, TreeList, Scheduler, Form, ScrollView, Popup, and others. The following issues may occur:
```

> - A component displays only a part of the content. There is no scrollbar to scroll down and reach the other part.

Reviewer: In here, "part" is used in its uncountable sense. We don't have strictly defined parts - it can be any part of content, just some volume of elements, but not all of them. So I wouldn't use an article.

I think if we mention the absence of scrollbar, the implications are fairly obvious (no scrollbar = no scrolling). So I'm shortening the description. This also helps me to avoid the phrases "there is"  and "scrollbar to scroll".

```suggestion
- A component displays only part of its content and has no scrollbar. 
```

> - You can scroll down the entire page but the component is not scrollable.

Reviewer: I don't see how this is different from the previous point. I read longer descriptions too. Same symptoms, same solutions.

> - A component height is `100%` but the component does not fit into the page.

Reviewer: ```suggestion
- Height is `100%` but the component does not fit into the page.
```

> - Scrolling methods do not work. For example, DataGrid does not navigate to a specific row when you call [navigateToRow](/Documentation/ApiReference/UI_Components/dxDataGrid/Methods/#navigateToRowkey) or enable [autoNavigateToFocusedRow](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#autoNavigateToFocusedRow).

Reviewer: When you say "methods", you mean programming interface, right? API? 

Methods can also mean different ways how users can scroll - for example, mouse wheel or keyboard shortcuts.

```suggestion
- Scrolling API does not work. For example, DataGrid does not navigate to the required row when you call [navigateToRow](/Documentation/ApiReference/UI_Components/dxDataGrid/Methods/#navigateToRowkey) or use [autoNavigateToFocusedRow](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#autoNavigateToFocusedRow).
```

> - A component displays a scrollbar but elements below the component overlap its content.

Reviewer: Not sure what you mean by "its content" - "scrollbar's content" or "component's content"?

> If the information below does not help you to enable scrolling and build the required layout, create a ticket in our Support Center and describe your issue in greater detail: [Create a ticket](https://supportcenter.devexpress.com/ticket/create).

Reviewer: ```suggestion
If information below does not resolve your scrolling-related issue, create a ticket in our Support Center and describe your scenario in detail: [Create a ticket](https://supportcenter.devexpress.com/ticket/create).
```
